### PR TITLE
SC-383: Add dependencies

### PR DESCRIPTION
**Pull request by xschildw**
This PR installs the dependencies found missing when manually installing R packages on an instance rebuilt from tag v2.1.3.

### CODE REVIEW

File: src/playbook.yaml [hunk 1]
```diff
@@ -47,16 +47,6 @@
           echo 'LoadModule python_module /usr/lib/apache2/modules/mod_python.so' >> /etc/apache2/mods-available/python.load
         chdir: /tmp/mod_python_source
 
-    - name: Install python3.8
```
Comment (zaro0508) [inline src/playbook.yaml:50]: why did you remove python 3.8 ?

Comment (xschildw) [reply src/playbook.yaml:50]: I think it's causing cloud-init to fail. As-is, the instance does not deploy (it times out and if I prevent the delete it is not correctly initialized (i.e. does not pass one of the reachability tests)).

Comment (zaro0508) [reply src/playbook.yaml:50]: that doesn't make sense because it does not fail on the v1.2.8 version which has python 3.10.12

**Comment by xschildw** [comment]
This branch is not needed anymore.
